app.json

```diff
@@ -15,6 +15,7 @@
     "runtimeVersion": {
       "policy": "sdkVersion"
     },
+    "jsEngine": "hermes",
```

Comment:
This is iOS only, right?

Reply:
as far as I know this is universal, but only became available for iOS in Expo 47. 
- https://hermesengine.dev/
- https://docs.expo.dev/guides/using-hermes/